apps/accounts/models/provider_request.py

```diff
@@ -50,6 +50,7 @@ class ProviderRequest(TimeStampedModel):
     created_by = models.ForeignKey(
         settings.AUTH_USER_MODEL, on_delete=models.CASCADE, null=True
     )
+    approved = models.DateTimeField(editable=False, blank=True, null=True)
```

Comment:
We currently show the approved status, but in the admin view, I couldn't see the time of approval listed in when visiting `/admin/accounts/providerrequest/5/change/`.

My guess is that it would make sense to add it near the status part of the admin view, as they seem linked to me.